.pytool/CISettings.py

```diff
@@ -51,10 +51,12 @@ def AddCommandLineOptions(self, parserObj):
 
     def RetrieveCommandLineOptions(self, args):
         super().RetrieveCommandLineOptions(args)
+
+        # In tree BaseTools are required for Rust build support so enable
+        # it by default.
+        self.UseBuiltInBaseTools = False
         if args.force_piptools:
             self.UseBuiltInBaseTools = True
-        if args.no_piptools:
```

Comment:
you should remove the -no_piptools argument if you are no longer checking for it

Reply:
I removed it. Just left an option to force pip tools.